### PR TITLE
update subworkflows and modules

### PR DESCRIPTION
**Pull request by ellisrichardj**
This PR updates the pre-installed modules (fastqc and multiqc) and subworkflows (utils_nextflow_pipeline, utils_nfcore_pipeline and utils_nfvalidation_plugin).  Default testing provides warnings when newer versions of modules and subworkflows are available, and following this updating is simple using the nf-core cli e.g. `nf-core modules update`

Some further updates were required to the nextflow script as the updated multiqc module expects additional inputs.  To get the tests passing these have been created as empty channels in the `workflows/genappdataanalysis.nf` script

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/APHA-CSU/genappdataanalysis/tree/master/.github/CONTRIBUTING.md)
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by Hira-Naveed — APPROVED** [review]
Great to see this coming along 🥇

**Review by praveenmurugan101 — APPROVED** [review]
Looks Great!

**Review by sainab-omar — APPROVED** [review]
Looks good 👍

File: subworkflows/nf-core/utils_nfcore_pipeline/main.nf [hunk 1]
```diff
@@ -34,7 +31,7 @@ workflow UTILS_NFCORE_PIPELINE {
 //  Warn if a -profile or Nextflow config has not been provided to run the pipeline
 //
 def checkConfigProvided() {
-    valid_config = true
+    def valid_config = true as Boolean
```
Comment (televisedbrain) [inline subworkflows/nf-core/utils_nfcore_pipeline/main.nf:34]: If valid_config absolutely has to be a boolean I'd declare like so
```nextflow
boolean valid_config = true
``` 
(No change required)

File: subworkflows/nf-core/utils_nfcore_pipeline/main.nf [hunk 2]
```diff
@@ -84,7 +83,7 @@ def workflowCitation() {
 // Generate workflow version string
 //
 def getWorkflowVersion() {
-    String version_string = ""
+    def version_string = "" as String
```
Comment (televisedbrain) [inline subworkflows/nf-core/utils_nfcore_pipeline/main.nf:86]: Same as above comment, if this has to be String I'd declare like 
```nextflow
String version_string = ""
```

**Review by televisedbrain — COMMENTED** [review]
Happy to approve but curious why the types have all been changed to dynamic

File: subworkflows/nf-core/utils_nfcore_pipeline/main.nf (hunk 1 above)
Comment (ellisrichardj) [reply subworkflows/nf-core/utils_nfcore_pipeline/main.nf:34]: Looks like this was to do with fixing syntax in the main nf-core repo: https://github.com/nf-core/modules/pull/6353 - I have just updated 🤷‍♂️

**Review by televisedbrain — APPROVED** [review]

**Pull request merged**